bootstrap.lisp

```diff
@@ -0,0 +1,25 @@
+;;;; Creates executable for linux and windows.
+;;; usage: sbcl --load bootstrap.lisp
+;;; emacs: M-! sbcl --load bootstrap.lisp
+
+(ql:quickload :aark)
+
+#+(and (not windows) (or win32 mswindows))(pushnew :windows *features*)
+
+(let* ((build-dir #+windows P"build/windows/" P"build/linux/")
```

Comment:
Hmm... If I understand, how `#+` works, on windows this line would become
 `(let* ((build-dir P"build/windows/" P"build/linux/")`, and that doesn't look like valid let binding for me.